api4/channel_test.go

```diff
@@ -4063,3 +4064,43 @@ func TestMoveChannel(t *testing.T) {
 		require.Equal(t, team2.Id, newChannel.TeamId)
 	}, "Should be able to (force) move private channel by a member that is not member of target team")
 }
+
+func TestRootMentionsMigrations(t *testing.T) {
```

Comment:
This test name doesn't look like what the test is actually testing. I think this should be something like `TestRootMentionsCount`.